docs/current_docs/install.mdx

```diff
@@ -7,3 +7,104 @@ import PartialInstallCli from './partials/_install-cli.mdx';
 # Installation
 
 <PartialInstallCli />
+
+## Update
+
+<Tabs>
+<TabItem value="macOS">
+
+Homebrew users can use the following commands:
+
+```shell
+brew update
+brew upgrade dagger
+```
+
+If you do not have Homebrew installed, you can use [install.sh](https://github.com/dagger/dagger/blob/main/install.sh):
+
+<CodeBlock language="shell">
```

Comment:
This needs importing (this is causing the docs build to fail).

Reply:
Added